cmd/aries-js-worker/vue-aries-framework-go/src/main.js

```diff
@@ -7,6 +7,9 @@ SPDX-License-Identifier: Apache-2.0
 import Vue from 'vue'
 import App from './App.vue'
 
+import { Aries } from "@hyperledger/aries-framework-go"
+Object.defineProperty(Vue.prototype, "$aries", { value: new Aries({}) })
```

Comment:
Might need to come back to this and see if this is the idiomatic way to do this. I had thought to use props.